src/EventSubscriber/HookEventBridge.php

```diff
@@ -18,6 +18,15 @@
  */
 final class HookEventBridge implements StaticEventSubscriber
 {
+	/**
+	 * This allows us to mock the Hook call in tests.
+	 *
+	 * @param string|array $data
+	 *
+	 * @return string|array
+	 */
+	private static $callHook = null;
```

Comment:
Shouldn't the DocBlock say the property is meant to be a `callable` or null? I also thought the property name could be more explicit about its finality,
```suggestion
	/**
	 * @var callable mockedCallHook
	 *
	 * @param string|array $data
	 *
	 * @return string|array
	 */
	private static $mockedCallHook = null;
```

Reply:
I fixed the code to `@var \Closure|null` and removed the param and return before reading your comment. I like the `$mockedCallHook` name.